demos/llm_npu/README.md

```diff
@@ -38,8 +38,10 @@ Run `export_model.py` script to download and quantize the model:
 
 **LLM**
 ```console
-python export_model.py text_generation --source_model meta-llama/Llama-3.1-8B-Instruct --target_device NPU --config_file_path models/config.json --model_repository_path models  --overwrite_models
+python export_model.py text_generation --source_model meta-llama/Llama-3.1-8B-Instruct --target_device NPU --config_file_path models/config.json --ov_cache_dir ./models/.ov_cache --model_repository_path models  --overwrite_models
```

Comment:
is there a requirement for existence of `ov_cache_dir` catalog or will it get created if not there?

Reply:
it will be created by the OV Runtime